src/React360Viewer.js

```diff
@@ -268,86 +268,86 @@ class React360Viewer extends Component {
         this.dragStart = null
     }
 
-    checkMobile(){
+    checkMobile() {
         this.isMobile = !!('ontouchstart' in window || navigator.msMaxTouchPoints);
     }
 
-    loadInitialImage(){
+    loadInitialImage() {
         this.currentImage = this.imageData[this.props.overrideImage || 0]
-        this.setImage()
+        this.setImage(true);
```

Comment:
Pass true to use the cached condition. Without this it always constructs a new canvas image and fetches from network cache which may or may not be ready by the time the user drags to that location.